.circleci/config.yml

```diff
@@ -35,13 +34,26 @@ jobs:
           paths:
             - dist
 
+  test-python: # Python tests are separated from YAML tests, because the latter take longer. This would allow Python tests to use any other available container and
+    docker:    # and yield it to another job, once finished, instead of counting only on the two containers assigned to the YAML tests.
```

Comment:
> Prefer a long one-line comment, or a multi-line comment _before_ the code, rather than spanning a line-end comment over several lines: the aim with line-end comments is to comment the line they are on 🙂